outputs/core_pools.json

```diff
@@ -59,10 +62,14 @@
     "0x9791d590788598535278552eecd4b211bfc790cb000000000000000000000498": "wstETH-WETH-BPT",
     "0xa899c63ba9ff275d6bfc83e43ee8e58ff6987584000200000000000000000594": "ECLP-rETH-wstETH",
     "0xb61371ab661b1acec81c699854d2f911070c059e000000000000000000000516": "ezETH/wstETH",
+    "0xb8cb384e65096386c1edaaf784e842c957fa3645000000000000000000000571": "inETH/wstETH",
     "0xc2598280bfea1fe18dfcabd21c7165c40c6859d30000000000000000000004f3": "wstETH/sfrxETH",
+    "0xc757f12694f550d0985ad6e1019c4db4a803f1600002000000000000000004de": "50wstETH/50sFRAX",
```

Comment:
Just over 100k.  Why was  this not picked up on #426 both tokens have been around for some time and have pricing.